src/server/game/Entities/Object/Updates/UpdateFieldFlags.cpp

```diff
@@ -2136,18 +2130,18 @@ uint32 UnitUpdateFieldFlags[PLAYER_END] =
     UF_FLAG_PRIVATE,                                        // PLAYER_FIELD_SUMMONED_BATTLE_PET_ID+1
     UF_FLAG_PRIVATE,                                        // PLAYER_FIELD_SUMMONED_BATTLE_PET_ID+2
     UF_FLAG_PRIVATE,                                        // PLAYER_FIELD_SUMMONED_BATTLE_PET_ID+3
-    UF_FLAG_PRIVATE,                                        // PLAYER__FIELD_KNOWN_TITLES
-    UF_FLAG_PRIVATE,                                        // PLAYER__FIELD_KNOWN_TITLES+1
-    UF_FLAG_PRIVATE,                                        // PLAYER__FIELD_KNOWN_TITLES+2
-    UF_FLAG_PRIVATE,                                        // PLAYER__FIELD_KNOWN_TITLES+3
-    UF_FLAG_PRIVATE,                                        // PLAYER__FIELD_KNOWN_TITLES+4
-    UF_FLAG_PRIVATE,                                        // PLAYER__FIELD_KNOWN_TITLES+5
-    UF_FLAG_PRIVATE,                                        // PLAYER__FIELD_KNOWN_TITLES+6
-    UF_FLAG_PRIVATE,                                        // PLAYER__FIELD_KNOWN_TITLES+7
-    UF_FLAG_PRIVATE,                                        // PLAYER__FIELD_KNOWN_TITLES+8
-    UF_FLAG_PRIVATE,                                        // PLAYER__FIELD_KNOWN_TITLES+9
-    UF_FLAG_PRIVATE,                                        // PLAYER__FIELD_KNOWN_TITLES+10
-    UF_FLAG_PRIVATE,                                        // PLAYER__FIELD_KNOWN_TITLES+11
+    UF_FLAG_PRIVATE,                                        // PLAYER_FIELD_KNOWN_TITLES
```

Comment:
Unless the names changed in client (and i think they arent in the client anymore) these are actual names so dont touch them